### PR TITLE
Add new package capa-explorer-web.vm

### LINKED ISSUE #1136: URL package proposal: capa-explorer-web.vm
### Package Name

capa-explorer-web

### Tool Name

capa Explorer Web

### Package type

OTHER/UNKNOWN

### Is the tool a console application?

false

### Version

0

### Category

Utilities

### Tool's authors

@s-ff, @mike-hunhoff, @williballenthin, @mr-tz

### Tool's description

capa Explorer Web is a web-based tool to explore the capabilities identified by capa.

### Download URL

https://mandiant.github.io/capa/explorer/capa-explorer-web.zip

### Download SHA256 Hash

TBD

### Dependencies

_No response_

### Why is this tool a good addition?

This tool allows you to interactively browse and display capa results in multiple viewing modes.

### Extra information

There is currently no version tracking. How do you recommend we adjust the URL/file to allow for easier tracking and updates?

This is similar to CyberChef so can be installed analogous most likely.

### PR DESCRIPTION
**Pull request by fariss**
closes #1136 

This PR adds a new VM-package capa-explorer-web.vm.

ZIP URL: `https://github.com/mandiant/capa/raw/refs/heads/master/web/explorer/releases/capa-explorer-web-v1.0.0-6a2330c.zip`

### CODE REVIEW

**Review by Ana06 — CHANGES_REQUESTED** [review]
@fariss thank you so much for providing a link for capa Explorer Web and for going even a step further and send a PR to add the package! :pray: The structure of VM-Packages is a bit tricky to learn and you have made an amazing job creating your first package! :blush: 

Thanks @sara-rn for reviewing the PR.

File: packages/capa-explorer-web.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  }
+  Install-ChocolateyZipPackage @packageArgs
+  VM-Assert-Path $toolDir
+
+  $chromePath = "${env:ProgramFiles}\Google\Chrome\Application\chrome.exe"
+  $capaWebPath = Get-Item "$toolDir\$toolName\*.html"
+  VM-Install-Shortcut -toolName $toolName -category $category -executablePath $chromePath -arguments "-home $capaWebPath"
+} catch {
+  VM-Write-Log-Exception $_
+}
```
[the remark this answers is not in the document]
Comment (fariss) [reply packages/capa-explorer-web.vm/tools/chocolateyinstall.ps1:26]: Thanks @Ana06 for the detailed analysis and follow-up enhancement PR! I did indeed look up `cyberchef.vm` before creating this package.

Just wanted to point out that the zip release has an inner folder called "capa-explorer-web" and inside that folder exists `index.html`.

[the remark this answers is not in the document]
Comment (Ana06) [reply packages/capa-explorer-web.vm/tools/chocolateyinstall.ps1]: The subfolder is common for many GH  related ZIPs (for example if you download a repo as ZIP), so we have the `innerFolder` argument to simplify these cases. :wink:

[the remark this answers is not in the document]
Comment (fariss) [reply packages/capa-explorer-web.vm/tools/chocolateyinstall.ps1]: Even better, thanks

File: packages/capa-explorer-web.vm/capa-explorer-web.vm.nuspec [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>capa-explorer-web.vm</id>
+    <version>1.0.0</version>
+    <authors>Mandiant FLARE</authors>
```
Comment (Ana06) [inline packages/capa-explorer-web.vm/capa-explorer-web.vm.nuspec:6]: Completely optional: @fariss I think you should be the author. The author in the nuspec is the tool author and you have implemented most of capa Explorer Web. :wink:

**Review by Ana06 — COMMENTED** [review]
@fariss thanks for the implementing the proposed changes! Could you squash your commits in just one `Add new package capa-explorer-web.vm`? I prefer to avoid having fixup commits to keep a clean commit history. When you do this, could you either remove me as co-author or change the email to my `anamg@google.com` email? The Google CLA check does not like my personal email.

If you need help to squash/modify the commits, please let me know and I can provide more information about how to do it. :wink:

**Comment by Ana06** [comment]
@mandiant/flare-vm I think we should add this package to the default config after we merge it :wink:

File: packages/capa-explorer-web.vm/capa-explorer-web.vm.nuspec (hunk 2 above)
Comment (fariss) [reply packages/capa-explorer-web.vm/capa-explorer-web.vm.nuspec]: Sounds good 👍, I've addressed these comments.

**Review by Ana06 — APPROVED** [review]
Tested locally and it works nicely, the code looks good too. Thanks @fariss for all the work!!! :heart_hands:

**Pull request merged**